mongodbatlas/resource_mongodbatlas_advanced_cluster_test.go

```diff
@@ -327,6 +332,13 @@ func TestAccClusterAdvancedCluster_PausedToUnpaused(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "paused", "false"),
 				),
 			},
+			{
+				Config:      testAccMongoDBAtlasAdvancedClusterConfigSingleProviderPaused(orgID, projectName, rName, true, "M10"),
```

Comment:
Could you also add a scenario where we test that is not possible to update a pause cluster? I think that here we are testing that the cluster cannot be set to pause again after a short period of time

Reply:
I believe the test you are referring is just above this one. You can see I have an `ExpectError` for `CANNOT_UPDATE_PAUSED_CLUSTER`

Reply:
I haven't seen it. Thanks! resolving this comment